src/Extension/CoreExtension.php

```diff
@@ -231,7 +231,7 @@ public function getFilters()
             new TwigFilter('lower', 'twig_lower_filter', ['needs_environment' => true]),
             new TwigFilter('striptags', 'strip_tags'),
```

Comment:
I suppose we have the same issue with this filter? And maybe all the other ones that call PHP's native function directly with no wrappers?

Reply:
I'll add the following:
* `sprintf`
* `strip_tags`

Should we also do something with `abs`? When null, pass 0?